Packs/DomainTools_Iris/Scripts/DomainToolsAddDomainRiskScoreToContext/add_domain_riskscore_to_context.py

```diff
@@ -0,0 +1,42 @@
+from CommonServerPython import *
+from typing import Dict, Any
+import traceback
+
+
+''' COMMAND FUNCTION '''
+
+
+def add_domain_riskscore_to_context(args: Dict[str, Any]) -> CommandResults:
+
+    domaintools_data = args['domaintools_data']
+
+    domain_name = domaintools_data.get("Name")
+    domain_risk_score = domaintools_data.get(
+        "Analytics", {}).get("OverallRiskScore")
+
+    result = {"Name": domain_name, "OverallRiskScore": domain_risk_score}
+
+    return CommandResults(
+        outputs_prefix='HighRiskPivotedDomains',
```

Comment:
Having a different root key than the name of the script can get confusing to users (who might see that key in context and be unsure which step of the playbook added it).
I think this will be a good way to make it more clear.

```suggestion
        outputs_prefix='AddDomainRiskScoreToContext.HighRiskPivotedDomains',
```

Reply:
updated!